### PR TITLE
Update Replacement Method on Example

### PR DESCRIPTION
**Pull request by joshw123**
Using the Example on the Documentation you get the following error 

`│ Error: Error updating region managed group instances: googleapi: Error 400: Invalid value for field 'resource.updatePolicy.maxSurge': '{  "percent": 20}'. maxSurge must be equal to 0 when replacement method is set to RECREATE, invalid`

I have updated the replacement method so that it can be copied directly

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [ ] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read [Write release notes](https://googlecloudplatform.github.io/magic-modules/contribute/release-notes/) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:note
Updating documentation update_policy block to working value
```

### CODE REVIEW

File: mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown [hunk 1]
```diff
@@ -174,10 +174,10 @@ update_policy {
   type                           = "PROACTIVE"
   minimal_action                 = "REPLACE"
   most_disruptive_allowed_action = "REPLACE"
-  max_surge_percent              = 20
-  max_unavailable_fixed          = 2
+  max_surge_fixed                = 3
+  max_unavailable_fixed          = 0
```
Comment (shuyama1) [inline mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown:178]:
```suggestion
  max_surge_fixed                = 0
  max_unavailable_fixed          = 2
```

File: mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown [hunk 2]
```diff
@@ -174,10 +174,10 @@ update_policy {
...
   min_ready_sec                  = 50
-  replacement_method             = "RECREATE"
+  replacement_method             = "SUBSTITUTE"
```
Comment (shuyama1) [inline mmv1/third_party/terraform/website/docs/r/compute_instance_group_manager.html.markdown:180]:
```suggestion
  replacement_method             = "RECREATE"
```

**Review by shuyama1 — COMMENTED** [review]
Sorry about the error in the example and thanks for making a fix PR! 

Only nit-pick: can we only change `max_surge_percent = 20` to `max_surge_fixed = 0` and leave others the way they are, since we have a test [`testAccRegionInstanceGroupManager_rollingUpdatePolicy3`](https://github.com/hashicorp/terraform-provider-google-beta/blob/a1000f3d5a33574f898d012b55853f2e7a744696/google-beta/resource_compute_region_instance_group_manager_test.go#L1245) configuring the resource with the same combination and therefore we can make sure it works or catch the error in our daily tests if it doesn't

**Comment by joshw123** [comment]
Hello @shuyama1 Yeah that works, thank you :)

**Review by shuyama1 — APPROVED** [review]

**Pull request merged**